src/store/redis.go

```diff
@@ -53,14 +52,11 @@ func (r *Redis) Connect() error {
 		Password: r.ConnectionInfo.Password,
 		Username: r.ConnectionInfo.Username,
 		DB:       0, // use default DB,
-		TLSConfig: &tls.Config{
-			MinVersion: tls.VersionTLS12,
-		},
```

Comment:
why? we need it to connect to the stuff in render o

Reply:
wow okay, i was having issues connecting the server to the redis port on docker that's why i removed that line.
i was getting `context deadline exceeded` error

i dont really think the specification of minimum version is required. i've connected to redis on render before without specifying that

Reply:
have you connected to the one I sent you without this? i need to ensure it's working

Reply:
true that. lemme try connecting with it

Reply:
so it didnt work :smiling_face_with_tear:  
pushing a fix. is okay if i add an env like `REDIS_TLS_ENABLED`?

Reply:
yes yes. let's do that

Reply:
https://github.com/odetolakehinde/slack-stickers-be/pull/13/commits/cd5b2117ce7b225e5ae9a92f87f4e7618c848edc